libsolidity/parsing/Token.h

```diff
@@ -303,10 +305,12 @@ class Token
 	static std::tuple<Token::Value, unsigned short, unsigned short> fromIdentifierOrKeyword(std::string const& _literal);
 
 private:
-	// extractM provides a safe way to extract numbers, 
+	// extractUnsigned provides a safe way to extract numbers, 
 	// if out_of_range error is thrown, they returns 0s, therefore securing 
-	// the variable's identity as an identifier.
-	static unsigned extractM(std::string const& _literal);
+	// the variable's identity as an identifier. If an invalid conversion
```

Comment:
Could you shorten this whole thing to `@returns 0 on error (invalid digit or number too large)` - this current text relies on internals about how exactly the conversion is done.